packages/backend/src/app.ts

```diff
@@ -224,21 +228,31 @@ export class App {
       resolvers
     })
 
-    // Setup Apollo on graphql endpoint
+    // Setup Apollo
     this.apolloServer = new ApolloServer({
       schema: schemaWithResolvers,
-      context: async (): Promise<ApolloContext> => {
-        return {
-          container: this.container,
-          logger: await this.container.use('logger')
-        }
-      }
+      plugins: [ApolloServerPluginDrainHttpServer({ httpServer })]
```

Comment:
Apollo highly recommends using this plugin. From Apollo docs:

```
We highly recommend using this plugin to ensure your server shuts down gracefully.

Specifically, it will:

    Stop listening for new connections
    Close idle connections (i.e., connections with no current HTTP request)
    Close active connections whenever they become idle
    Wait for all connections to be closed
    After a grace period, if any connections remain active, forcefully close them.
```

Although, do we still need it since we have graceful shutdown for the container ?